### DISCUSSION TITLE
Consume and generate TailwindCSS configurations and classes

### DISCUSSION THREAD

**Discussion by janbaykara** [concrete-utopia/utopia#1391] [Ideas]
This is a quick one. My first thought at finding this project was: wow, this looks really cool!

My second thought was: could this integrate with something like TailwindCSS, to generate and consume design tokens which makes the code more resilient beyond the tool?

**Comment by maltenuhn** [comment]
Hi there! I know it might look like we favour css-in-js, but we actually used Tailwinds to build our website. It's a pretty amazing system! As it stands, we already support adding css classes (and the inspector will show you if a property is set coming from css). Having better support for it would be nice though, and I dream of being able to convert from tailwinds to css-in-js _and vice versa_. The last part is probably a lot harder though :D 

More to your point - we'll be looking a lot at real-life styling options over the summer. That includes both better support for styledComponents / Emotion, themes, layout components, and yes, utility classes. We'll share ideas and designs in Discord and here as this progresses.

**Reply by wokalski** [reply]
If you can already add/remove classes that's already good enough if you ask me!

**Reply by janbaykara** [reply]
Thanks for the reply! So cool that you can inspect the properties of class names. Look forward to seeing how this progresses.
